### ISSUE TITLE
File handle get exhausted when submission is failed

### ISSUE THREAD

**Issue by Vimos** [nocaps-org/updown-baseline#3]
Just in case you forgot to add a correct token, the submission won't succeed and your system may get stuck because of handle leaking.

This might be caused by: 

https://github.com/nocaps-org/updown-baseline/blob/9b5b567bcf16863ba8c9b31bc1d34728151e97f0/updown/utils/evalai.py#L78

I think we can change it to 
```
-        json.dump(predictions, open(predictions_filename, "w"))
+        with open(predictions_filename, "w") as f:
+            json.dump(predictions, f)
```

**Comment by kdexd** [comment]
Yes, this makes sense. Thanks a lot for pointing this out! Could you make this change and open a PR? I will be happy to merge it!

**Comment by kdexd** [comment]
Fixed by #4
